src/config/indexer-config.c

```diff
@@ -0,0 +1,142 @@
+/*
+ * Wazuh Indexer Configuration
+ * Copyright (C) 2015, Wazuh Inc.
+ * August 31, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#ifndef CLIENT
+#ifndef WIN32
```

Comment:
Is this guard necessary?

Reply:
That's why the #ifndef CLIENT is included, but my question was about this one #ifndef WIN32. Anyway, it was just a question, no changes required.

Reply:
I also think that #ifndef WIN32 is unnecessary, but I keep the same guards as in other files of config directory.